cmd/kubebuilder/generate/generate.go

```diff
@@ -92,6 +92,28 @@ func doGen(g string) bool {
 }
 
 func RunGenerate(cmd *cobra.Command, args []string) {
+	root, err := os.Executable()
+	if err != nil {
+		glog.Fatalf("error: %v", err)
+	}
+	root = filepath.Dir(root)
+
+	if len(args) > 0 && args[len(args)-1] == "core-controller-only" {
+		if doGen("controller-gen") {
+			c := exec.Command(filepath.Join(root, "controller-gen"),
```

Comment:
I am still learning this codebase but would like to explore if we don't have to invoke a new binary to do this. Can `controller-gen` codebase be part of `kb` itself ?

Reply:
nvm, looking at the code below .. it looks more consistent with other generators :)